bayesian/utils.py

```diff
@@ -42,6 +54,10 @@ def server_create_analysis(ecosystem, package, version, api_flow=True, force=Fal
     :param force_graph_sync: force synchronization to graph
     :return: dispatcher ID handling flow
     """
+    # Bookkeeping first
+    component = MavenCoordinates.normalize_str(package) if ecosystem == 'maven' else package
+    server_create_component_bookkeeping(ecosystem, component, version, user_profile)
+
     args = {
         'ecosystem': ecosystem,
         'name': MavenCoordinates.normalize_str(package) if ecosystem == 'maven' else package,
```

Comment:
@fridex I am not repeating the call to server_create_component_bookkeeping. There are two scenarios for which server_create_component_bookkeeping is called separately.
1.  When component data is available on graphdb.
2. When component data is not available, componentApiFlow is called before calling bayesianFlow or bayesianApiFlow.

Did I miss anything here?